lib/tasks/import_events.rake

```diff
@@ -35,6 +35,8 @@ namespace :import do
           event_row_hash[key] = csv_event_row[event_hash[key]]&.strip || ""
         elsif [:start_date, :end_date].include? key
           event_row_hash[key] = csv_event_row[event_hash[key]]&.strip || nil
+          valid_date = Date.parse(event_row_hash[key]) rescue nil
+          Rails.logger.info "Invalid date! for #{event_row_hash[:title]}" if valid_date.nil?
```

Comment:
You could use `unless valid_date` instead; if `Date.parse` is successful, `valid_date` will contain a date, if not, it will be nil, so just doing `unless valid_date` is enough